modules/hostap/src/supp_api.c

```diff
@@ -1406,6 +1406,59 @@ int hapd_config_network(struct hostapd_iface *iface,
 out:
 	return ret;
 }
+
+int supplicant_ap_config_params(const struct device *dev, struct wifi_ap_config_params *params)
+{
+	struct hostapd_iface *iface;
+	const struct wifi_mgmt_ops *const wifi_mgmt_api = get_wifi_mgmt_api(dev);
+	int ret = 0;
+
+	if (params->type & WIFI_AP_CONFIG_PARAM_MAX_INACTIVITY) {
+		if (!wifi_mgmt_api || !wifi_mgmt_api->ap_config_params) {
+			wpa_printf(MSG_ERROR, "ap_config_params not supported");
+			return -ENOTSUP;
+		}
+
+		ret = wifi_mgmt_api->ap_config_params(dev, params);
+		if (ret) {
+			wpa_printf(MSG_ERROR,
+					   "Failed to set maximum inactivity duration for stations");
```

Comment:
Alignment is wrong here, make sure the string starts in the same column as the `MSG_ERROR` in previous line.